lib/ex_ice/priv/ice_agent.ex

```diff
@@ -559,40 +642,39 @@ defmodule ExICE.ICEAgent.Impl do
     gath_trans = Map.new(gath_trans)
 
     if stale_gath_trans != [] do
-      Logger.debug("Gathering transactions timed out: #{inspect(Keyword.keys(stale_gath_trans))}")
+      ids = Enum.map(stale_gath_trans, fn {id, _} -> id end)
```

Comment:
`Map.keys/1`? Not sure if this is a map.

Reply:
It's a list of tuples